lms/envs/aws.py

```diff
@@ -724,7 +724,7 @@
         days=ENV_TOKENS.get('OAUTH_EXPIRE_PUBLIC_CLIENT_DAYS', OAUTH_EXPIRE_PUBLIC_CLIENT_DAYS)
     )
     OAUTH_ID_TOKEN_EXPIRATION = ENV_TOKENS.get('OAUTH_ID_TOKEN_EXPIRATION', OAUTH_ID_TOKEN_EXPIRATION)
-
+    OAUTH_DELETE_EXPIRED = ENV_TOKENS.get('OAUTH_DELETE_EXPIRED', True)
```

Comment:
Should we be setting OAUTH_DELETE_EXPIRED = True in common.py so more things inherit it, and then using that setting as the default in aws.py (like we do for OAUTH_ID_TOKEN_EXPIRATION)
This way tests/devstacks would be more likely to have the delete behavior that prod has.